### ISSUE TITLE
Totals on stacked bar chart

### ISSUE THREAD

**Issue by philipcdavis** [masayuki0812/c3#984]
It would be nice to be able to add totals as a data label to stacked bar charts and to their corresponding tool tips. Something like [this](http://www.highcharts.com/demo/column-stacked).

This might require additional design decisions like exposing the ability to center data labels within bars charts as well.

**Comment by masayuki0812** [comment]
Yeah, looks nice. Let me mark as enhancement and plugin candidate. Thanks!

**Comment by avoliva** [comment]
Was this ever implemented?

**Comment by smartameer** [comment]
Is it possible to show only the total instead of cluttering all values? Lets say my stacked bar values are [0, 1, 2, 500] and in this case values overlap each other for 0, 1, 2 with formatting like 'n degree'.  Is it possible to sum everything and show just one at the top.

**Comment by nbpalomino** [comment]
Bump...

Some news about this?

**Comment by johnnymetz** [comment]
+1 showing the total on top of the stacked bar chart would be very useful

**Comment by ArtyomMinsk** [comment]
Any updates on showing total on the top?

**Comment by n2lose** [comment]
Any news about the optional display text label center in the bar? I got same problem.

**Comment by tvw** [comment]
Here a quick solution on how to get totals on stacked bar charts:

You can make a stacked bar chart a combination chart by adding a line into the chart which has as data the totals. Because the line represents the totals, it sits exactly on top of the bars, because the stack size is the sum=total. Now we use CSS to remove the line, because you want only see the labels and in order to avoid overlapping labels, we move the labels of the total line a little bit up via CSS. That's it.

Here is a fiddle, which shows the result: [https://jsfiddle.net/thomasw/t2aj7xsL/20/](https://jsfiddle.net/thomasw/t2aj7xsL/20/)

**Comment by guptavijay208** [comment]
Any updates on showing total on the top, I also need this feature urgently?

**Comment by Zenthae** [comment]
sorry to dig out such an old subject but any news about it ? i do need this feature.

**Comment by apolinario** [comment]
While this does not get implemented, the person that made this fiddle hacked their way to make it work, might be useful for someone else as it was for me: https://jsfiddle.net/Dimmy/d5sxjng2/6/
